lib/stripe/request.ex

```diff
@@ -144,6 +144,20 @@ defmodule Stripe.Request do
 
   def get_id!(_), do: raise("You must provide an ID or a struct with an ID to this operation.")
 
+  @spec prefix_expansions(t, binary) :: list
+  def prefix_expansions(%Request{opts: opts} = request, prefix) do
```

Comment:
Would this be needed if the request looked like this?  I.e. it could be handled by your awesome work in `add_object_expansion`?

`Stripe.Charge.retrieve("ch_123", [expand: "balance_transaction"])`

For multiple, here is one option.  We would need to split on `,` to generate `expand[]=balance_transaction&expand[]=application_fee`.
`Stripe.Charge.retrieve("ch_123", [expand: "balance_transaction,application_fee"])`

Reply:
Note I think `expanded: ["balance_transaction"]` is ruby? whereas `opts` in our case is a keyword list (https://elixir-lang.org/getting-started/keywords-and-maps.html)...

Side note - I also noticed a doc block that could be updated.

Reply:
The `prefix_expansions` needs to happen before it makes it into `add_object_expansion` because it's only applicable to the list function on the resource and there is no context of this once we reach `API.request`.

I'm not sure on the advantage of using a comma separated value over a list here as we're just splitting it back out into a list as soon as it's needed. Also this would make it slightly more involved for users who want to build this list dynamically as they now need to worry about joining it correctly as well.

> Note I think expanded: ["balance_transaction"] is ruby?

I'm not sure I follow here. The syntax in my examples are adding the `:expand` key into the existing `opts` keyword list.

Reply:
Oh I was looking at the PR desc w/o the braces around `expanded`.  I see in your test you meant for it to be a proper keyword list! 👍